external/flume-sink/pom.xml

```diff
@@ -85,6 +97,13 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <configuration>
+          <relocations combine.self="override" />
+        </configuration>
```

Comment:
is it worth adding a short comment here explaining why this is necessary?  maybe this is obvious to others more versed in maven, but it was a little mysterious to me.  Then again we don't doc a lot of the intricacies of the build system.  anyway, just a suggestion, I'll leave it up to your judgement.